tensorflow/contrib/tensorrt/convert/convert_graph.cc

```diff
@@ -126,6 +126,14 @@ bool IsTensorRTCandidate(const tensorflow::Node* node) {
     "Prod",
     "Max",
     "Min",
+    "Relu6",
+#endif
+#if NV_TENSORRT_MAJOR >= 5
+    "QuantizeV2",
```

Comment:
FYI, I don't think we will ever have QuantizeV2 and Dequantize in the input graph. Of course, doesn't hurt to support them.

Reply:
Are these ops no longer supported in favor of QuantizeAndDequantizeV2/3? I noticed there is no python API to add QuantizeAndDequantizeV2/3 nodes.

Reply:
Though our initial example used the QunatizeV2/Dequantize ops, going forward our models will use only QuantizeAndDequantizeV2/3. The last example already uses the new combined op.